dev-client/src/util.ts

```diff
@@ -63,7 +63,7 @@ export const formatPercent = (value: number) => {
 };
 
 export const formatCoordinate = (value: number) => {
-  return value.toLocaleString(undefined, {
+  return value?.toLocaleString(undefined, {
```

Comment:
I don't think this is correct. The `value` parameter should be declared as optional. This will also return `undefined` for a zero coordinate value, which seems incorrect since zero is a valid lat/lon value.

Reply:
value is not optional here. but if it's undefined, I don't want to try to call `.toLocaleString()` and crash.

This works fine for `0`. Optional chaining only fails on undefined or null, not other falsy values.

Did I misunderstand?